### ISSUE TITLE
Will not work?

### ISSUE THREAD

**Issue by willb08** [rohitbatra1/Safari-Reverse-Image-Search#2]
Well I downloaded the extension and for some reason I don't follow why it what propagate the link search once right clicked or two finger tapping on a image. I get the contextual box but no propagation. So I am here looking for assistance in understanding why and perhaps help in remediating the issue.

[Screen Recording 2020-07-31 at 2.15.12 PM.zip](https://github.com/rohitbatra1/Safari-Reverse-Image-Search/files/5008686/Screen.Recording.2020-07-31.at.2.15.12.PM.zip)

**Comment by rohitbatra1** [comment]
Hi,

So there is actually a known design conflict with StopTheMadness which another user experienced as well.

(STM attempts to prevent websites from messing with or disabling the default contextual menu, so, to my knowledge, it disables the javascript which passes the information from the selected image to the context menu handler in the extension.)

**Fix**: If you go into the settings for StopTheMadness and go into website options, there should be a box for contextual menus which if you disable will allow the context menu options of my extension to work.

Also, if you prefer to leave that setting enable, holding down the option key while opening the contextual menu temporarily overrides StopTheMadness.

Please let me know if either of these options do not fix the issue.

**Comment by willb08** [comment]
Yes, that was the culprit. I did as you suggested and I disabled the contextual menu and it worked like a charm.
I guess I have to be more intuitive when it comes to overriding which competing scripts have priority. it would have been a good minute to figure out which textual code was causing the issue. Thank you so very much for your help and expertise. Great work on your extension, I think you again for your knowledge and the ability to add more to my ubiquitous flow in my work through a simple extension that you have provide, keep up the great work.
